docker-compose.yml

```diff
@@ -334,6 +336,104 @@ services:
       volumes:
         - ./apps/collections/log:/app/log
 
+  publisher: &publisher
+      build: apps/publisher
+      depends_on:
+        - publishing-api
+        - rummager
+        - publisher-worker
+        - diet-error-handler
+        - calendars
```

Comment:
I don't think Publisher depends on Calendars.

Reply:
We were unable to get it to create new artefacts due to it wanting to pull in the bank/public holidays (seemed to be related to fact check - https://github.com/alphagov/publisher/blob/b1078b616dcf99901a2625111521ad496b7d7469/app/views/noisy_workflow/request_fact_check.text.erb)

Reply:
Ah, my mistake!